chart/README.md

```diff
@@ -10,4 +10,75 @@ This repository offers a convenience chart for the `kube-downscaler` [project](h
 | arguments          | list   | `[--interval=60,--include-resources=deployments,statefulsets,horizontalpodautoscalers,scaledobjects]` | Arguments to pass to the kube-downscaler binary |
 | excludedNamespaces | list   | `["namespace-a", "namespace-b"]`                                                                      | Namespaces to exclude from downscaling          |
 
-How the downscaler can be configured is described in the [kube-downscaler documentation](https://codeberg.org/hjacobs/kube-downscaler).
+
+
+# Deploy Kube-downscaler using Helm chart
+
+This directory contains tutorial to deploy Kube-downscaler.
+
+## Configuring your Deployment to downscale
+
+Please add below annotations based on timezone your deployment should run:
+```
+metadata:
+  annotations:
+    downscaler/uptime: "Mon-Fri 07:00-19:00 US/Eastern"
+```
+Note: For more configuration details please, refer [here](https://github.com/hjacobs/kube-downscaler#configuration).
+
+## Architecture
+The diagram below depicts how a Kube-downscaler agent control applications.
+![Alt text](images/architecture.png?raw=true "Kube Kube-downscaler diagram")
+
+## Quick Start
+Below are instructions to quickly install and configure Kube-downscaler.
+
+### Installing Kube-downscaler
+
+1. Make sure connected to right cluster:
+```
+kubectl config current-context
+```
+2. Set right environment depending on cluster:
+```
+export KDS_ENV='[minikube | testing | staging | production]'
+```
+3. Before deploy make sure to update *values.yaml* in Kube-downscaler chart depending on your cluster support for RBAC:
+```
+rbac:
+  create: false
+```
+Note: In case RBAC is active new service account will be created for Kube-downscaler with certain privileges, otherwise 'default' one will be used.
+
+4. Deploy Kube-downscaler:
+```
+helm install . --values "config/${KDS_ENV}.yaml" --namespace default  --name kube-downscaler
```

Comment:
We shouldn't encourage people to deploy in the `default` namespace. Let's either remove the namespace or use a more appropriate value for the occasion, like `kube-downscaler`. This command also requires the user be in the `chart` folder.

Reply:
yes, will call it py-kube-downscaler